Source branch changes

javadev opened the pull request.

MehmetAzizYirik commented:
@javadev in the index.html we have some sentences starting with "Functionality is provided for many areas in cheminformatics including:" and it follows render InChI etc. We must remove them since they are not related to maygen project.

javadev commented:
@MehmetAzizYirik 

I added background and fixed texts.

![Screenshot_127](https://user-images.githubusercontent.com/348496/143831203-256d5e8e-8b03-4e47-a89f-cfc16915485c.png)
